lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -33,22 +33,25 @@ class HighChart
     #
     def to_html(placeholder=random_canvas_id)
       chart_hash_must_be_present
+      script = load_dependencies('web_frameworks')
```

Comment:
I will look into the Proxy design. However, `to_html` will load the dependencies only if there is some additional dependency is provided in the `modules` option or if a map is being drawn (to load the required map data). Otherwise, `load_dependencies` will simply return an empty string.

Reply:
`to_html` will load the dependencies only if there is some additional dependency provided in the modules option or if a map is being drawn (to load the required map data). Otherwise, load_dependencies will simply return an empty string. I will look into the Proxy design pattern also.

Reply:
Need to handle this.